app/models/Nino.scala

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
can we put this in a domain package?

Reply:
or even a registration, as we only use the nino in that case, I dont want the Nino getting mixed in with VOA models.